OCAT/client/resources/js/components/shared/services/login.service.js

```diff
@@ -0,0 +1,37 @@
+import axios from "axios";
+// from https://loizenai.com/reactjs-jwt-authentication-example/#implement-reactjs-jwt-authentication-service
+
+export class LoginService {
+    static async submit (login) {
+        try {
+            await axios.post('http://localhost:4567/api/login/submit', login).then(response => {
+                localStorage.setItem("isSupervisor", JSON.stringify(response.data));
+                localStorage.setItem("isLogedIn", true);
+
+                window.location.replace("http://localhost:4567/assessment/list");
+              })
+        }
+        catch (err) {
+            throw new Error(`${err.response.statusText} - ${err.response.data.message}`);
+        }
+    }
+
+    static async logout() {
+        try {
+            localStorage.removeItem("isSupervisor");
+            localStorage.removeItem("isLogedIn");
+            
+            window.location.replace("http://localhost:4567/api/logout");
+        }
+        catch (err) {
+            throw new Error(`${err.response.statusText} - ${err.response.data.message}`);
+        }
+    }
+
+    static getCurrentUser() {
```

Comment:
Providing local storage to the front-end components, was I able to provide data from the session instead?